content/security/index.adoc

```diff
@@ -4,48 +4,60 @@ title: "Jenkins Security"
 section: security
 ---
 
+The Jenkins project takes security seriously.
+We make every possible effort to ensure users can adequately secure their automation infrastructure.
```

Comment:
It is fine to split sentences by logic sections (e.g. by commas) with a tab for the next lines. Looks well. I am mostly against the case when people just align the stuff to feet the border (e.g. 120 symbols) and start sentences without line break. But it's my IMHO anyway